src/main/java/org/jboss/tools/intellij/openshift/tree/application/ChangeActiveProjectLinkNode.java

```diff
@@ -0,0 +1,25 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.intellij.openshift.tree.application;
+
+import com.redhat.devtools.intellij.common.tree.LinkElement;
+import org.jboss.tools.intellij.openshift.actions.project.ChangeActiveProjectAction;
+
+public class ChangeActiveProjectLinkNode extends MessageNode<NamespaceNode> implements LinkElement {
+    protected ChangeActiveProjectLinkNode(ApplicationsRootNode root, NamespaceNode parent) {
+        super(root, parent, "Missing project, <a>choose different</a> or create one");
```

Comment:
The current impl in common cannot handle 2 links currently. Ok, for changing the wording, I was using the same as in vscode.

Reply:
I'm rewording to only mention "choose different" (ommitting "create new"). I'd like to avoid "click here" as the jetbrains design guide tells us to avoid it
https://jetbrains.design/intellij/controls/link/
<img width="693" alt="image" src="https://github.com/redhat-developer/intellij-openshift-connector/assets/25126/eedc1a98-c1d4-4f79-a35d-f17ebdd1f286">